pkg/clients/elasticache/elasticache_test.go

```diff
@@ -988,11 +986,39 @@ func TestCacheClusterNeedsUpdate(t *testing.T) {
 			},
 			want: false,
 		},
+		{
+			name: "NeedsNoUpdateNormalizedMaintenanceWindow",
+			kube: replicationGroupParams(func(p *v1beta1.ReplicationGroupParameters) {
+				p.PreferredMaintenanceWindow = aws.String("Mon:00:00-Fri:23:59")
+
+			}),
+			cc: elasticachetypes.CacheCluster{
```

Comment:
was looking at building a helper for this, but `elasticachecluster_test.go` already defines helpers for the CacheCluster struct in the same package, which would potentially collide with the needs of these tests. Would like some guidance on the best way to deal with this